### PR TITLE
added more robust testing for admin endpoints

### PR DESCRIPTION
**Pull request by RyderBlues**
## Type of change
- [x ] New feature (non-breaking change which adds functionality)

## Change Status
- [x ] Complete, tested, ready to review and merge

# Checklist
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by dylan-rinella** [comment]
Code looks very clean, cant find anywhere that I can critic negatively. Looks great.

File: __tests__/routes/adminIncidents.test.js [hunk 1]
```diff
@@ -20,6 +20,18 @@ describe('[GET] /dashboard/incidents', () => {
...
+    let pendingIncidents = 0;
+    for (let i = 0; i < res.body.length; i++) {
+      if (res.body[i].status == 'pending') {
+        pendingIncidents += 1;
+      }
+    }
+
+    expect(res.body.length).toBe(pendingIncidents);
+  });
 });
```
Comment (dylan-rinella) [inline __tests__/routes/adminIncidents.test.js:36]: Love this approach.

**Review by devbrunopaula — APPROVED** [review]
👍 🥇

**Review by joseph-witzke — APPROVED** [review]
Looks good!

**Pull request merged**